marketplace/deployer_util/schema_values_common.py

```diff
@@ -16,31 +16,31 @@
 
 import config_helper
 
-CODEC_ASCII = 'ascii'
-CODEC_UTF8 = 'utf_8'
 
+VALUES_FILE = {
+  'stdin': '-',
+  'raw': '/data/values.yaml',
+  'expanded': '/data/final_values.yaml',
+}
 
-def add_to_argument_parser(parser,
-                           values_file='/data/final_values.yaml',
-                           values_dir='/data/final_values'):
-  parser.add_argument(
-      '--values_file',
-      help='Yaml file to read values from. Takes precendence '
-      'over --values_dir if the file exists. '
-      'Use "-" to specify reading from stdin',
-      default=values_file)
-  parser.add_argument(
-      '--values_dir', help='Where to read value files', default=values_dir)
-  parser.add_argument(
-      '--values_dir_encoding',
-      help='Encoding of --values_dir file contents',
-      choices=[CODEC_UTF8, CODEC_ASCII],
-      default=CODEC_UTF8)
+VALUES_DIR = {
+  'stdin': '/dev/null',
+  'raw': '/data/values',
+  'expanded': '/data/final_values',
+}
+
+
+def add_to_argument_parser(parser):
   parser.add_argument(
       '--schema_file',
       help='Path to the schema file',
       default='/data/schema.yaml')
 
+  parser.add_argument(
+      '--mode',
```

Comment:
`values_mode` to make it clear. Especially in `provision.py`, `--mode` is very ambiguous.

Reply:
Done!